src/test/resources/expected-classes/jackson-resolver-union/without-model-package/MyResultObjectSuffix.java.txt

```diff
@@ -2,8 +2,10 @@
     value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
     date = "2020-12-31T23:59:59-0500"
 )
+@com.fasterxml.jackson.annotation.JsonTypeInfo(use = com.fasterxml.jackson.annotation.JsonTypeInfo.Id.NAME, property = "__typename")
+@com.fasterxml.jackson.databind.annotation.JsonTypeIdResolver(GraphqlJacksonTypeIdResolver.class)
```

Comment:
Correct me if I'm wrong but looking at the test `generate_CheckFiles_without_model_package_and_with_prefix_and_suffix()` that generates this file there is no package defined that's why there isn't a fully-qualified package name defined here.

Reply:
Yeah, and I think it is an issue.

Reply:
Sorry I´m confused. 

On that test the `GraphqlJacksonTypeIdResolver.java` is generated as well without package and since there is no package you can´t use the fully-qualified name.

And if we add to that test a package the tests no longer makes sense because as far I understand this test is to test when there is no package.

I´m happy to do any change required but I´m not really sure what needs to be done here.

Reply:
Yes, that's what I am talking about. The existing tests are false positives.
But never mind. I will take care of it in a separate pull request.
Yours is good to go!🚀 
Thanks for the contribution!

Reply:
Oh understood. That's why I finding it confusing.

I will be making more contributions in the future.